sources/background_scripts/notifications.js

```diff
@@ -0,0 +1,25 @@
+NewTabTweaker = {};
+NewTabTweaker.Notifications = {
+	notification_ids: [],
+	notify: function(notification_id, message){
+		var title = browser.runtime.getManifest().name;
+		
+		// Do not display the same message twice by using a notification_id
+		browser.notifications.create(notification_id,
+		{
+			type: "basic",
+			iconUrl: browser.extension.getURL("/icons/main_64.png"),
```

Comment:
The value of `iconUrl` should be declared as a `const` variable outside the function, as per the same reasoning for line 5.